strongbox-web-core/src/main/java/org/carlspring/strongbox/security/authentication/strategy/NpmLoginAuthenticationStrategy.java

```diff
@@ -19,26 +19,26 @@
 import java.util.regex.Matcher;
 
 @Component
-public class NpmLoginAuthenticationSupplier
-        extends LayoutAuthenticationSupplier
+public class NpmLoginAuthenticationStrategy
+        extends LayoutAuthenticationStrategy
 {
 
     @Inject
     private ObjectMapper objectMapper;
 
-    public NpmLoginAuthenticationSupplier()
+    public NpmLoginAuthenticationStrategy()
     {
         super(NpmLayoutProvider.ALIAS);
     }
 
     @CheckForNull
     @Override
-    public Authentication supply(@Nonnull HttpServletRequest request)
+    public Authentication convert(@Nonnull HttpServletRequest request)
     {
         NpmUser npmUser = deserializeNpmUser(request);
 
         if (!isValidNpmUser(npmUser) ||
-            !usernamesMatch(request.getRequestURI(), npmUser.getName()))
+                !usernamesMatch(request.getRequestURI(), npmUser.getName()))
```

Comment:
Cosmetic: Please, follow our coding convention:
```suggestion
           !usernamesMatch(request.getRequestURI(), npmUser.getName()))
```

Reply:
Done